File: HackerRank - Diagonal Difference/solver.py
#!/bin/python3

import math
import os
import random
import re
import sys
#
# Complete the 'diagonalDifference' function below.
#
# The function is expected to return an INTEGER.
# The function accepts 2D_INTEGER_ARRAY arr as parameter.
#


def diagonalDifference(arr):
    # Write your code here
    lr = 0
    rl = 0
    for i in range(len(arr)):
        for j in range(len(arr[i])):
            if i == j:
                lr += arr[i][j]
            if i + j == len(arr) - 1:
                rl += arr[i][j]
    return abs(lr - rl)


if __name__ == '__main__':
    fptr = open(os.environ['OUTPUT_PATH'], 'w')

    n = int(input().strip())

    arr = []

    for _ in range(n):
        arr.append(list(map(int, input().rstrip().split())))

    result = diagonalDifference(arr)

    fptr.write(str(result) + '\n')

    fptr.close()

File: Paragon - Hiring Test/Problem1/solver.ts
function rotasi_matriks(n: number, m: number, matrik: number[][]): number[][] {
    // Your code starts here.
    const result: number[][] = [];

    for (let i = 0; i < m; i++) {
        const arr: number[] = [];
        for (let j = n - 1; j >= 0; j--) {
            arr.push(matrik[j][i]);
        }
        result.push(arr);
    }

    return result;
}

export default rotasi_matriks;
